relaxr-assignment-2-3-t-oskanian/javascript/scripts.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(function() {
+$('.read-more, .less-content').hide();

```

Comment:
@toskan similar to HTML, be mindful of indentation when writing your functions, make sure the curly brackets are being lined up/close up appropriately.